netty/src/main/java/io/grpc/netty/NettyClientStream.java

```diff
@@ -60,14 +60,17 @@
   private Metadata headers;
   private final Channel channel;
   private final NettyClientHandler handler;
-  private AsciiString authority;
   private final AsciiString scheme;
+  private final AsciiString userAgent;
+  private AsciiString authority;
+
   private Http2Stream http2Stream;
   private Integer id;
   private WriteQueue writeQueue;
 
   NettyClientStream(MethodDescriptor<?, ?> method, Metadata headers, Channel channel,
-      NettyClientHandler handler, int maxMessageSize, AsciiString authority, AsciiString scheme) {
+      NettyClientHandler handler, int maxMessageSize, AsciiString authority, AsciiString scheme,
+      AsciiString userAgent) {
```

Comment:
@carl-mastrangelo refresh my memory ... do we allow the app to override userAgent on a per-call basis?